### ISSUE TITLE
Not stable delta time in default update() method on macOS Mojave

### ISSUE THREAD

**Issue by DenisSmirnov1986** [cocos2d/cocos2d-x#19364]
- cocos2d-x version: Cocos-2dx 3.17.1
- devices test on: MacBook Pro (Retina, 13-inch, Early 2015)
- developing environments
   - Xcode version: 10.1
   - macOS Mojave 10.14.2

Issue is almost the same as [this](https://github.com/cocos2d/cocos2d-x/issues/17806),
except that the problem is on macOS Mojave.
Delta time that passed to default `update()` method is not stable.
I created a simple HelloWorld project and added `scheduleUpdate()` to the main scene, then I logged `delta` like this:
```
void HelloWorld::update(float delta) {
    log("delta = %f", delta);
}
```
Here is an example of received values:
```
delta = 0.017894
delta = 0.015337
delta = 0.016745
delta = 0.016337
delta = 0.016656
delta = 0.017666
delta = 0.016186
delta = 0.016146
delta = 0.018187
delta = 0.017124
delta = 0.016165
delta = 0.018551
delta = 0.017515
delta = 0.016795
delta = 0.016660
```
Note that some values even bigger than 0.018 sec.
It seems to me that the problem started since I updated to Mojave a week ago.
My game works not smoothly because of this issue and it is critical for me.

**Comment by DeRealMorgan** [comment]
I am not totaly sure about this: I think this is no issue. The delta time is supposed to give you the time difference between the last and the current frame. If it takes longer sometimes (you compute different things or your computer is busy) that will change the delta time.

**Comment by Anywhere-Music-Player** [comment]
Yes, for Mojave I have the same issue now. My game is shuttering.
But I'm building for iOS only, so I'm happy at least it was fixed for it.
